### ISSUE TITLE
Support for badges

### ISSUE THREAD

**Issue by mathebox** [openHPI/xikolo-ios#592]
**Is your feature request related to a problem? Please describe.**
The platform can have badges (file format: png) additionally to the normal certificates.

**Describe the solution you'd like**
Let's have an additional "certificate" for the badge in the certificate course area.

**Additional context**
Currently the data for badges is not included in the API. At this point the app can only handle PDF certificates

**Comment by janrenz** [comment]
its not just a png, its a https://de.wikipedia.org/wiki/Open_Badges

**Comment by cwillems** [comment]
For the handling in the app, the PNG metadata doesn't make a real difference. The badges are however meant for sharing (FB, Twitter, LinkedIn, ...) - supporting the user here would be the real advancement.

**Comment by mathebox** [comment]
Yes, I forgot to mention that a share sheet should also be available. Right now it's already possible to share the other certificates via the app :)
